src/NavBar/NavBar.scss

```diff
@@ -0,0 +1,43 @@
+﻿.navBar {
```

Comment:
Yesterday we discussed how to avoid conflicts with CSS names.

perhaps something like

```css
.navBar {
    // stuff
    
    .logo {
        // stuff
    }

    // more things
}
```